src/client/sandboxes/code-instrumentation/location.js

```diff
@@ -0,0 +1,95 @@
+import UrlUtil from '../../utils/url';
+import createPropertyDesc from '../../utils/create-property-desc.js';
+import { GET_LOCATION_METH_NAME, SET_LOCATION_METH_NAME } from '../../../processing/js/index';
+import { isLocation } from '../../utils/types';
+
+class LocationWrapper {
```

Comment:
I get it now. Let's move classe to the separate files. Always try to follow the rule: one class - one file. So we will have:
`location/index.js`, `location/wrapper.js`